src/cloud/paths/provision.yml

```diff
@@ -0,0 +1,59 @@
+put:
+    operationId: PutProvision
+    tags:
+      - Provision
+    description: Create a new user, organization, and bucket. Idempotent. Requests can be retried.
+    requestBody:
+      description: Create user and org with supplied IDs
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/ProvisionRequest.yml"
+    responses:
+      "201":
+        content:
+          application/json:
+            schema:
+              $ref: "../schemas/ProvisionResponse.yml"
+        description: The user name and ID, the org name and ID, the token for user if requested
+      "400":
+        content:
+          application/json:
+            schema:
+              $ref: "../../common/responses/ServerError.yml"
+        description: Bad request. Data in request is invalid. Examples include Invalid IDs, invalid names, and invalid limits
+      "401":
+        content:
+          application/json:
+            schema:
+              $ref: "../../common/responses/ServerError.yml"
+        description: Credentials not provided
+      "403":
+        content:
+          application/json:
+            schema:
+              $ref: "../../common/responses/ServerError.yml"
+        description: Insufficient credentials to create an organization
+      "409":
+        content:
+          application/json:
+            schema:
+              $ref: "../../common/responses/ServerError.yml"
+        description: Conflict. Causes can be user already exists with a different name, org already exists with a different name, org exists with different limits.
```

Comment:
Will we also get a 409 when an org name is already in use with a different ID? If not, can/should we?

Reply:
Although, maybe that should be 409?

Reply:
Turns out, due to the conversion of internal codes to Http codes, that I cannot return a 409 at all. Any conflict type will be a 422. I will amend the document